modules/javafx.graphics/src/main/java/com/sun/marlin/MaskMarlinAlphaConsumer.java

```diff
@@ -105,28 +101,22 @@ public void setMaxAlpha(int maxalpha) {
     static final OffHeapArray ALPHA_MAP_UNSAFE_NO_AA;
 
     static {
-        final Unsafe _unsafe = OffHeapArray.UNSAFE;
-
         // AA:
         byte[] _ALPHA_MAP = buildAlphaMap(MarlinConst.MAX_AA_ALPHA);
         ALPHA_MAP = _ALPHA_MAP; // Keep alive the OffHeapArray
         ALPHA_MAP_UNSAFE = new OffHeapArray(ALPHA_MAP, ALPHA_MAP.length); // 1K
 
-        long addr = ALPHA_MAP_UNSAFE.address;
-
         for (int i = 0; i < _ALPHA_MAP.length; i++) {
-            _unsafe.putByte(addr + i, _ALPHA_MAP[i]);
+            ALPHA_MAP_UNSAFE.putByte(i, _ALPHA_MAP[i]);
```

Comment:
I know you are trying to keep the diffs to a minimum, but since there is no more Unsafe, can we replace the "UNSAFE" with something more meaningful?

Reply:
Thanks for the reminder. I actually do want to change this. I had originally put a "FIXME" comment to do that, but removed it during the cleanup prior to the draft PR.

My thought was to use something like "OFF_HEAP" rather than a bake the implementation of the off-heap memory into the name.